samples/openthread/cli/README.rst

```diff
@@ -105,7 +105,7 @@ The following snippets are available:
   Not compatible with the ``tcat`` snippet.
 
   .. note::
-    When building with the ``multiprotocol`` snippet for the ``nrf5340dk/nrf5340/cpuapp`` build target, the :kconfig:option:`SB_CONFIG_NETCORE_MULTIPROTOCOL_RPMSG` Kconfig option has to be set to ``y``.
+    When building with the ``multiprotocol`` snippet for the ``nrf5340dk/nrf5340/cpuapp`` build target, the :makevar:`FILE_SUFFIX` CMake variable has to be set to ``ble``.
```

Comment:
```suggestion
    When building with the ``multiprotocol`` snippet for the ``nrf5340dk/nrf5340/cpuapp`` board target, set the :makevar:`FILE_SUFFIX` CMake variable to ``ble``.
```
I believe we need to say **board** target instead of build target in the new HWMv2 world, see https://github.com/nrfconnect/sdk-nrf/pull/15213

Reply:
Yes, `build_target` is something else, as clarified with @tejlmand . Hence my PR.